neuro_san/client/streaming_input_processor.py

```diff
@@ -92,6 +94,13 @@ def process_once(self, state: Dict[str, Any]) -> Dict[str, Any]:
             returned_sly_data: Dict[str, Any] = self.processor.get_sly_data()
             origin_str = Origination.get_full_name_from_origin(self.processor.get_answer_origin())
 
+        # Update the sly_data if new sly_data was returned
+        if returned_sly_data is not None:
+            if sly_data is not None:
+                sly_data.update(returned_sly_data)
+            else:
+                sly_data = returned_sly_data.copy()
```

Comment:
... or copying the returned sly_data dictionary if we didn't have sly_data before.